packages/bpk-component-modal/src/BpkModal.js

```diff
@@ -24,14 +24,14 @@ import { withScrim } from 'bpk-scrim-utils';
 import { Portal, cssModules } from 'bpk-react-utils';
 
 import STYLES from './BpkModal.scss';
-import BpkModalDialog, {
+import BpkModalInner, {
   propTypes as modalDialogPropTypes,
   defaultProps as modalDialogDefaultProps,
   type Props as ModalDialogProps,
-} from './BpkModalDialog';
+} from './BpkModalInner';
```

Comment:
Renamed `BpkModalDialog` to `BpkModalInner` now that we have separated dialog and modal out